# Epigenomics
#Phase II
 - What is being done in the experiment, hypothesis, procedure and design of experiment.  
 Data sets 2 and 3 are similar. Human embryonic stem (hES) cells (WA09, also called H9) were exposed to a panel of chemicals at 
 different concentrations for 24 hours.  
 The chemicals are known to exert effects on histone methylation or acetylation namely HATi, HDACi, HDACa, HDMTi, HMTi and include some
 putative inactive controls. 
 We choose doses that produced ~90-100% and ~50-70% viability to sequence by RNA-Seq.  
 We’re looking for patterns of gene expression to derive a gene signature predictive of epigenetic responsiveness.  
 Of course, many of the gene expression changes can be downstream from the original target genes that are affected by changes in histone 
 post-translational modifications.  
 The goal is to develop a ChIP-based assay detecting direct effects of chemicals on histone modifications, 
 so ultimately, we’ll need to refine the gene signature by using ChIP-Seq to identify the original target genes of chemical exposure.  
 Our backup plan is to use an RNA-based screening assay that would rely on a predictive signature simply based on common effects of 
 epigenotoxicant chemicals on gene expression.  
 Either way, we’ll focus our attention on genes most relevant to cell differentiation and organ development.
- What stage is the experiment in.
The phase I grant used iPS cells, a few chemicals and ChIP-PCR (arrays). 
The current phase, phase II grant is using human ES cells, a large panel of chemicals, RNA-Seq , and ideally, ChIP-Seq.
- Source of data  
The Set  2 and 3 data sets are RNA-Seq data.

